Specs/DateTime/German/MergedExtractor.json

```diff
@@ -1 +1,144 @@
-[]
\ No newline at end of file
+[{
+    "Input": "das sind 2 Tage",
+    "NotSupportedByDesign": "python",
+    "NotSupported":"javascript",
+    "Results": [
+      {
+        "Text": "2 Tage",
+        "Type": "duration"
+      }
+    ]
+  }
+  ,
+  {
+    "Input": "es ist 4 Uhr",
+    "NotSupportedByDesign": "python",
+    "NotSupported":"javascript",
+    "Results": [
+      {
+        "Text": "4 Uhr",
+        "Type": "time"
+      }
+    ]
+  }
+  ,
+  {//should be datetime
+    "Input": "Das ganze findet dann am Montag um 4 Uhr statt",
+    "NotSupportedByDesign": "python",
+    "NotSupported":"javascript",
+    "Results": [
+      {
+        "Text": "am Montag um 4 Uhr ",
+        "Type": "set"
+      }
+    ]
+  },
+  {
+    "Input": "Wie sieht denn dieser Tag so aus?",
+    "NotSupportedByDesign": "python",
+    "NotSupported":"javascript",
+    "Results": [
+      {
+        "Text": "dieser Tag",
+        "Type": "date"
+      }
+    ]
+  },
+  {
+    "Input": "letzte Woche",
+    "NotSupportedByDesign": "python",
+    "NotSupported":"javascript",
+    "Results": [
+      {
+        "Text": "letzte Woche",
+        "Type": "daterange"
+      }
+    ]
+  },
+  {//should be datetimerange
+    "Input": "Das meeting geht morgen von 12 bis 14 Uhr.",
+    "NotSupportedByDesign": "python",
+    "NotSupported":"javascript",
+    "Results": [
+      {
+        "Text": "morgen von 12 bis 14 Uhr.",
+        "Type": "set"
+      }
+    ]
+  },
+  {
+    "Input": "Verschieb mal das meeting vom 22. August auf den 23. August",
+    "NotSupportedByDesign": "python",
+    "NotSupported":"javascript",
+    "Results": [
+      {
+        "Text": "22. August",
+        "Type": "date"
+      },
+      {
+        "Text": "23. August",
+        "Type": "date"
+      }
+    ]
+  },
+  {
+    "Input": "06/06 12:15",
+    "NotSupportedByDesign": "python",
+    "NotSupported":"javascript",
+    "Results": [
+      {
+        "Text": "06/06 12:15",
+        "Type": "datetime"
+      }
+    ]
+  },
+  {
+    "Input": "06/06/12 15:15",
+    "NotSupportedByDesign": "python",
+    "NotSupported":"javascript",
+    "Results": [
+      {
+        "Text": "06/06/12 15:15",
+        "Type": "datetime"
+      }
+    ]
+  },
+  {
+    "Input": "29. Mai",
+    "NotSupportedByDesign": "python",
+    "NotSupported":"javascript",
+    "Results": [
+      {
+        "Text": "29. Mai",
+        "Type": "date"
+      }
+    ]
+  },
+  {
+    "Input": "Ich wurde im August geboren",
+    "NotSupportedByDesign": "python",
+    "NotSupported":"javascript",
+    "Results": [
+      {
+        "Text": "August",
+        "Type": "daterange"
+      }
+    ]
+  },
+  {
+    "Input": "Das nächste meeting wird am 16. März 2017 sein. Wie wäre es denn mit einer Diskussionsrunde heute um 2 Uhr Nachmittags?",
+    "NotSupported": "javascript",
+    "NotSupportedByDesign": "python",
+    "Results": [
+      {
+        "Text": "16. März 2017 ",
+        "Type": "set" //should be date
```

Comment:
If this is wrong, this test should not be enabled. Should be moved to the correct spec and marked as notsupported for now.